cmd/scaleway-cloud-controller-manager/main.go

```diff
@@ -46,6 +48,9 @@ func main() {
 	}
 
 	fs := pflag.NewFlagSet("scaleway-cloud-controller-manager", pflag.ContinueOnError)
+	klogFlags := goflag.NewFlagSet("klog", goflag.ContinueOnError)
```

Comment:
ExitOnError maybe?

Reply:
ExitOnError or ContinueOnError, it's not copied when added to fs `fs.AddGoFlagSet(klogFlags)`